### PR TITLE
Added Percona Knobs documentation

### PR DESCRIPTION
**Pull request by dbpercona**

### CODE REVIEW

File: docs/knobs.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+Configuration File:
+```
+Percona:
+  ignoreLongIndexError: {true|false}
+```
+
+Default value: true 
+
+Older versions of MongoDB would insert and update documents even if an index
```
Comment (denis-protyvenskyi) [inline docs/knobs.md]: @dbpercona the description should be the opposite:
Older versions of MongoDB wouldn't insert and update documents if an index key was too long. By setting `failIndexKeyTooLong` to `false` MongoDB will insert or update the documents, but won't index keys if they are too long (so index lookup won't find such documents, although full collection scan without index usage will return the document).

Comment (denis-protyvenskyi) [inline docs/knobs.md]: My bad, everything is correct here.

File: docs/knobs.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+--failIndexKeyTooLong {true|false}
+```
+
+Configuration File:
+```
+Percona:
+  ignoreLongIndexError: {true|false}
+```
+
+Default value: true
```
Comment (igorsol) [inline docs/knobs.md]: I think the default for `ignoreLongIndexError` is `false` while the default for command line option is `true`. This corresponds to their names: one is `ignore` and other is `fail`.

**Comment by dbpercona** [comment]
We are canceling knobs.   I will rename this for setParameter documentation.

**Pull request closed without merging**